lib/active_merchant/billing/gateways/sage_pay.rb

```diff
@@ -405,14 +442,20 @@ def post_data(action, parameters = {})
         parameters.update(
           Vendor: @options[:login],
           TxType: TRANSACTIONS[action],
-          VPSProtocol: @options.fetch(:protocol_version, '3.00')
+          VPSProtocol: @protocol_version
```

Comment:
I'm just curious why you decided to use add_override_protocol_version to update the version instead of just updating this line to be something like options[:protocol_version] || @protocol_version?

Reply:
`post_data` does not get the `options` hash, nor does the method that calls it (`commit`) so rather than pass in options to two additional methods and update all the `commit` invocations it felt a little simpler to add the method. I'm open to either way though!

Reply:
Gotcha, makes sense!